source/patterns/00-atoms/block/image/image.twig

```diff
@@ -1,12 +1,14 @@
 {%- import 'atoms-html' as html -%}
 
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['image']) }) -%}
+
 <picture {{- html.attributes(attributes|default({})) }}>
 
   {%- for source in sources|default([]) -%}
     <source {{- html.attributes(source) }}>
   {%- endfor -%}
 
-  {%- set image = image|default({})|merge({class: image.class|default([])|merge(['image']) }) -%}
+  {%- set image = image|default({})|merge({class: image.class|default([])|merge(['image__image']) }) -%}
```

Comment:
Suggests this pattern should be renamed `picture`, then the top-level class becomes `picture` and this class becomes `picture__image`.

Reply:
`image_img`? `picture` is more precise if you're thinking about this as representing HTML elements, which they aren't (this one just happens to). I think I would definitely refer to this an an 'image' rather than a 'picture'...